app/src/main/res/values/strings.xml

```diff
@@ -30,27 +30,27 @@
   <string name="done">Done</string>
   <string name="enter">Enter</string>
   <string name="cancel">Cancel</string>
-  <string name="hint">Prompt</string>
-  <string name="help_create_db_path">It is recommended that you use cloud paths such as Dropbox and WebDAV. This will auto-sync your database to the cloud, making it easy to use the database on other devices. \n The database is encrypted using AES-256. Without your password, no one can open your database.</string>
-  <string name="help_pass_type">Use password only: \n Only your password is needed to open the database；\n Password + key: \n You need to enter a password and select a key file to open the database.</string>
-  <string name="help_pass_key">A key file is equivalent to a key to open data, and is usually much more complex and reliable than a password. But it is also more difficult to hide. If you keep the database in the cloud, do not put the key with the database there. \n You can choose any file as a key to open the database, a photo, or a TXT file. \n⚠️ Note: If this photo or TXT document is modified, you will not be able to open the database! !! \n It is highly recommended to use read-only files as keys.</string>
+  <string name="hint">Notice</string>
+  <string name="help_create_db_path">Using cloud services like Dropbox and WebDAV is recommended. This auto-syncs your database to the cloud, making it easy to use it on other devices. \n The database is encrypted using AES-256. This means your passowrd is needed to open it.</string>
+  <string name="help_pass_type">Password only:\nThe password alone can open the database；\n Password+key:\nA An additional keyfile must also be selected open the database.</string>
+  <string name="help_pass_key">A keyfile is a file used as a database password because it has more data. It is more difficult to hide and can not be uploaded to the cloud if your database is.\n⚠️ Any file can be used, but will stop working if the file is modified!\n Only use read-only files as keys.</string>
```

Comment:
```suggestion
  <string name="help_pass_key">A keyfile is a file used as a database password because it has more data. It is more difficult to hide and can not be uploaded to the cloud if your database is.\n⚠️ Any file can be used, but will stop working if the file is modified!\n Only use read-only files as keyfiles.</string>
```